_maps/map_files/generic/CentCom.dmm

```diff
@@ -10016,6 +10022,12 @@
 	heat_capacity = 1e+006
 	},
 /area/centcom/tdome/observation)
+"Dz" = (
+/obj/machinery/smartfridge/chemistry/virology/preloaded{
+	pixel_x = 0
+	},
```

Comment:
This won't do anything, and shouldn't exist (someone else will have to clean it up later). Did you run the Git Hooks?

https://hackmd.io/@tgstation/SyVma0dS5#Git-Hooks

Reply:
Wait, is pixel_x set on the fridge itself codeside?

Reply:
Nope, I just edited it via https://fastdmm2.ss13.io/